Readme.md

```diff
@@ -104,6 +104,9 @@ When _elm-make_ encounters a compile error, we keep _elm-live_ running and give
 #### `--pushstate`
 Serve `index.html` on 404 errors. This lets us use client-side routing in Elm. For instance, we can have a URL like `http://localhost:8000/account` get handled by the Elm _navigation_ package instead of failing with a 404 error.
 
+#### `--before-build`
+Run a command before a build is run. This allows us to call external build tools that could generate Elm code, for example.
```

Comment:
Any opinions on how the proof-of-concept notice should be formatted? Or worded, for that matter.